docs/blog/contributor-development-maintenance-guide.en-US.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Contributor development maintenance guide
+date: 2023-03-10
+author: kiner-tang
+---
+
+Hi, my name is [kiner-tang](https://github.com/kiner-tang). My personal work is closely related to Ant Design, so I can be considered a heavy user of Ant Design.It is for this reason that I am slowly working my way up from a user to a Contributor, contributing some of the problems I encountered and the new features I concluded to the community, and finally being honored to become a member of Ant Design Collaborator.
+
+During the journey from user to contributor, and then from contributor to collaborator, I also encountered many problems. I would like to take this opportunity to sort out and summarize these problems, hoping to be helpful to new contributors and collaborators joining Ant Design community.
+
+## Some frequently asked Questions
+
+### About dependent version
+
+Popular package managers, whether npm, yarn, or pnpm, offer version-locking solutions that, most of the time, help us avoid version-inconsistencies in packages.However, in Ant Design projects, many functions depend on the original components in the `react-component` repository. We expect that when we encounter some bugs, after the `react-component` fixes and releases the patch version, Instead of manually upgrading the version in the Ant Design project, we can install the latest patch installation package only by reinstalling the dependencies.At this point, the version-lock file provided by the package manager is the main obstacle to automatic updates, because once the version-lock file is available, reinstalling the dependency will also install the version specified by the lock file and cannot be upgraded to the patch version.
```

Comment:
There are various of package mangers, such as ..., providing version-locking solutions to avoid version-inconsistencies issue.

Reply:
many functions depend on the underlying dependencies of `react-component`.

Reply:
We hope Ant Design follow the rule of semver. When underlying dependencies fix the bug, user can easily reinstall the `node_modules` to fix instead care about too many detail about internal packages.

Reply:
done